pkg/archive/archive.go

```diff
@@ -256,8 +269,16 @@ func blobInArchive(file string) string {
 }
 
 func includeFile(fpath string) bool {
+	includeFiles := map[string]struct{}{
+		config.InternalDir:         {},
+		"catalogs":                 {},
```

Comment:
Just a nit. This random string among other constants is really ugly to see. At some point, it should be a variable/constant somewhere so it won't be so lonely here.

Reply:
Totally agree. It's set as a const in the `cli` package, but I will move it to `config` so it can be referenced like the others.